invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -101,16 +92,14 @@ const defaultRecord = {
 // However, it is RDMDepositForm that gets to decide what are the frontend
 // defaults for this datamodel
 function defaultize(initialRecord) {
-  let record = { ...defaultRecord };
-  record.metadata = { ...record.metadata, ...initialRecord.metadata };
-  return record;
+  return { ...defaultRecord, ...initialRecord };
```

Comment:
I am not sure this will work because the defaults from the backend for titles / creators / contributors were not right and it was easier to fix them on the frontend. This will take the backend ones... But things may have changed since!

Reply:
We tested it and it seems working! What was before there, was causing problems...we should definitely remove all these transformations....